pages/dkp/kommander/2.2/backup-and-restore/index.md

```diff
@@ -76,15 +80,117 @@ data:
         #   aws_secret_access_key=<REDACTED>
 ```
 
-After you have created the `ConfigMap` with **MinIO**, OR **Amazon S3**, patch the Velero `AppDeployment` by adding the `configOverrides` value. This applies the `ConfigMap` to your instance after the cluster has been configured:
+### Azure Blob
 
-```bash
-cat << EOF | kubectl -n kommander patch appdeployment velero --type="merge" --patch-file=/dev/stdin
-spec:
-  configOverrides:
-    name: velero-overrides
-EOF
-```
+Prerequisites: Create Azure related assets such as storage account, blob containers, resource group, roles, service principals prior to continuing.
```

Comment:
We should elaborate a bit here or at least provide links to relevant Azure docs.